### ISSUE TITLE
change volum lable on Windows

### ISSUE THREAD

**Issue by Alex6357** [openzfsonwindows/openzfs#264]
now I can't change volume label of a dataset (default full name of the dataset) in windows and something like myzpool/myzfs is ugly. Is it possible to add a property representing the volume lable?

**Comment by lundman** [comment]
Ah yep, could perhaps be done. I wonder if just a user property would work, like `windows:volume_label`.

**Comment by lundman** [comment]
Although, you can probably set `zfs set mountpoint=Games myzpool/myzfs`. Then using `driveletter` to decide if you want it as a driveletter, or mounted inside the parent.

Although I do not remember if the volume name I present is based on the mountpoint.

**Comment by Alex6357** [comment]
Nope. It's still the name of the dataset. And mountpoint must be an absolute path. If I set `zfs set mountpoint=/games myzpool/myzfs` and set `zfs set driveletter=G myzpool/myzfs` , the volume label will still be `myzpool/myzfs`.
